Gemfile.lock

```diff
@@ -181,7 +181,7 @@ GEM
     concurrent-ruby (1.0.5)
     crack (0.4.3)
       safe_yaml (~> 1.0.0)
-    crass (1.0.4)
+    crass (1.0.6)
```

Comment:
I was curious what this was and it turns out that it's a CSS parser written entirely in ruby. Pretty neat: https://github.com/rgrove/crass